pywbem/_tupletree.py

```diff
@@ -160,8 +160,8 @@ def xml_to_tupletree_sax(xml_string, meaning, conn_id=None):
     # 1. xml.sax.parseString() raises UnicodeEncodeError for unicode strings
     #    that contain any non-ASCII characters (despite its Python 2.7
     #    documentation which states that would be supported).
-    # 2. The SAX parser in Python 2.6 and 3.4 (pywbem does not support 3.1 -
-    #    3.3) does not accept unicode strings, raising:
+    # 2. No longer a reason: The SAX parser in Python 2.6 and 3.4 does not
```

Comment:
Why not just get rid of the whole item 2.

Reply:
DONE